tests/AppComponent.js

```diff
@@ -0,0 +1,4 @@
+baseline: "#baseline",
+countryPackageInfo: "#country-package-info",
+firstResultLink: '#search-results :first-child a',
+firstResultName: '#search-results :first-child .name',
```

Comment:
Why bothering adding an extra `class="name"` instead of just using `#search-results h4`, or even `ul h4` ?